modules/sdk-core/src/bitgo/staking/iStakingWallet.ts

```diff
@@ -26,6 +26,31 @@ export interface StakeOptions {
 export interface UnstakeOptions {
   amount: string;
   clientId?: string;
+  delegationId?: string;
+}
+
+export interface DelegationOptions {
+  delegationStatus?: string;
+  delegationIds?: Set<string>;
+  page?: number;
+  pageSize?: number;
+  sortBy?: string;
```

Comment:
I think for `sortBy` we need type as well, otherwise not sure if it's intuitive what is accepted 

```
    CREATED_DATE_DESC("-createdDate"),
    CREATED_DATE_ASC("createdDate"),
    ``